### PR TITLE
Refactor: Calling batch input

### PR DESCRIPTION
**Pull request by mbtools**
Standardizes calling batch input API and handling errors.

### CODE REVIEW

**Comment by flaiker** [comment]
:+1:

By the way: Do `zif_abapgit_gui_jumper` and `zcl_abapgit_gui_jumper` maybe better belong to the UI package?

**Comment by mbtools** [comment]
Maybe `z.._abapgit_object_jumper` is a better name because the dependencies are within the `objects*` packages 😉  

There's also this https://github.com/abapGit/abapGit/issues/3990. Eventually, we should get a clearer separation layers:
UI, repo, git, and objects, for example, and a well defined interface for the object layer.

File: src/ui/zif_abapgit_gui_jumper.intf.abap [hunk 1]
```diff
@@ -23,4 +26,12 @@ INTERFACE zif_abapgit_gui_jumper
     RAISING
       zcx_abapgit_exception.
 
+  METHODS jump_batch_input
+    IMPORTING
+      !iv_tcode      TYPE sy-tcode
+      !it_bdcdata    TYPE ty_bdcdata_tt
+      !iv_new_window TYPE abap_bool DEFAULT abap_true
```
Comment (larshp) [inline src/ui/zif_abapgit_gui_jumper.intf.abap:33]: this variable is not used?

**Review by larshp — CHANGES_REQUESTED** [review]

File: src/ui/zif_abapgit_gui_jumper.intf.abap (hunk 1 above)
Comment (mbtools) [reply src/ui/zif_abapgit_gui_jumper.intf.abap:33]: it's used in `zcl_abapgit_object_dial->delete` (the only non-jump place using batch input)

Comment (larshp) [reply src/ui/zif_abapgit_gui_jumper.intf.abap:33]: yea, its supplied, but not used in the method implementation?

Comment (mbtools) [reply src/ui/zif_abapgit_gui_jumper.intf.abap:33]: argh. yes, forgot the other call without `starting new task`...

**Review by larshp — APPROVED** [review]

**Pull request merged**